09-gis.Rmd

```diff
@@ -618,87 +618,82 @@ In addition, if you would like to run simulations with the help of a geodatabase
-One can easily access these tools via the command line. 
-In the code chunk below `linkGDAL()` searches the computer for a working GDAL installation and adds the path of the executable files to the PATH variable. 
-This allows us to easily run GDAL via system calls.
-For example, `ogrinfo` gives back meta information of vector data.
+As discussed in Chapter \@ref(read-write), GDAL is a low-level library that supports many spatial data formats.
+GDAL is so effective that most GIS programs use GDAL in the background for importing and exporting geographic data, rather than re-inventing the wheel and using bespoke read-write code.
+But GDAL offers more than data I/O.
+It has [geoprocessing tools](http://www.gdal.org/pages.html) for vector and raster data, functionality to create [tiles](https://www.gdal.org/gdal2tiles.html) for serving raster data online, and rapid [rasterization](https://www.gdal.org/gdal_rasterize.html) of vector data, all of which can be accessed via the system of R command line.
+
+The code chunk below demonstrates this functionality:
+`linkGDAL()` searches the computer for a working GDAL installation and adds the location of the executable files to the PATH variable, allowing GDAL to be called on different operating systems.
+In the example below `ogrinfo` provides metadata above a vector dataset:
 
 ```{r, eval=FALSE, message=FALSE}
 link2GI::linkGDAL()
-cmd = paste("ogrinfo -ro -so -al", 
-            system.file("shape/nc.shp", package = "sf"))
+cmd = paste("ogrinfo -ro -so -al", system.file("shape/nc.shp", package = "sf"))
 system(cmd)
-#>INFO: Open of `C:/Users/geocompr/Documents/R/win-library/3.5/sf/shape/nc.shp'
-#>    using driver `ESRI Shapefile' successful.
-#>
-#>Layer name: nc
-#>Metadata:
-#> DBF_DATE_LAST_UPDATE=2016-10-26
-#>Geometry: Polygon
-#>Feature Count: 100
-#>Extent: (-84.323853, 33.881992) - (-75.456978, 36.589649)
-#>Layer SRS WKT:
-#>GEOGCS["GCS_North_American_1927",
-#>    DATUM["North_American_Datum_1927",
-#>        SPHEROID["Clarke_1866",6378206.4,294.9786982]],
-#>    PRIMEM["Greenwich",0],
-#>    UNIT["Degree",0.017453292519943295],
-#>    AUTHORITY["EPSG","4267"]]
-#>AREA: Real (24.15)
-#>PERIMETER: Real (24.15)
-#>CNTY_: Real (24.15)
-#>CNTY_ID: Real (24.15)
-#>NAME: String (80.0)
-#>FIPS: String (80.0)
-#>FIPSNO: Real (24.15)
-#>CRESS_ID: Integer (9.0)
-#>BIR74: Real (24.15)
-#>SID74: Real (24.15)
-#>NWBIR74: Real (24.15)
-#>BIR79: Real (24.15)
-#>SID79: Real (24.15)
-#>NWBIR79: Real (24.15)
-```
-
-This is of course a very simple example, and we could have achieved the same using `rgdal::ogrInfo()`.
-Still, it shows how we can use GDAL via the command-line from within R independently of other packages, and nothing stops us from using more advanced and/or not already implemented GDAL utilities in R.
+#> INFO: Open of `C:/Users/geocompr/Documents/R/win-library/3.5/sf/shape/nc.shp'
```

Comment:
thanks for shortening the long, long, long, long output (Jakub has also already criticized this)

Reply:
Haha no problem

Reply:
Shortened a little more now :+1:

Reply:
:+1: